modules/serverless-logic-workflow-base-builder-image.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-logic/serverless-logic-creating-managing-workflows.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-logic-workflow-base-builder-image_{context}"]
+= Configuring the workflow base builder image
+
+If your scenario requires strict policies for image usage, such as security or hardening constraints, you can replace the default image used by the {ServerlessLogicOperatorName} to build the final workflow container image.
```

Comment:
I have added a note about this in the description. 
{ServerlessLogicOperatorName} will not reflect right now because I have added this attribute in the (https://github.com/openshift/openshift-docs/pull/76488). Once we merge that PR this attribute will reflect correctly.